src/s3/commands/uploadFile.ts

```diff
@@ -17,6 +22,16 @@ import { readablePath } from '../util'
 import { progressReporter } from '../progressReporter'
 import { localize } from '../../shared/utilities/vsCodeUtils'
 import { showErrorWithLogs, showOutputMessage } from '../../shared/utilities/messages'
+import { TextEditor } from 'vscode'
```

Comment:
Duplicate import. Use `vscode.TextEditor` since we're already importing all of `vscode`